chia/full_node/mempool.py

```diff
@@ -395,6 +400,8 @@ def create_bundle_from_mempool_items(
         log.info(f"Starting to make block, max cost: {self.mempool_info.max_block_clvm_cost}")
         with self._db_conn:
             cursor = self._db_conn.execute("SELECT name, fee FROM tx ORDER BY fee_per_cost DESC, seq ASC")
+        skipped_items = 0
+        STANDARD_XCH_SPEND_COST = 6_000_000
```

Comment:
good idea to give this a name. I think it would make sense to put it at the top with the other constants and also give it a comment that this determines how close to the block size limit we're willing to go, basically